app/views/content_items/service_manual_guide.html.erb

```diff
@@ -1,11 +1,73 @@
-<h1>
-  <%= @content_item.title %>
-</h1>
+<!-- Breadcrumb -->
+<!-- Proposed component:'govuk_component/breadcrumb' -->
+<div class="govuk-breadcrumb">
+  <ol>
+    <li><a href="/">Service Manual</a></li>
+    <li><a href="/">Topic</a></li>
+    <li>Guide name</li>
+  </ol>
+</div>
 
-<ul class="header-links">
-  <% @content_item.header_links.each do |header_link| %>
-    <li><a href="<%=header_link[:href]%>"><%=header_link[:title]%></a></li>
-  <% end %>
-</ul>
+<!-- Page title and contact -->
+<div class="grid-row">
+  <div class="column-two-thirds">
 
-<%= @content_item.body.html_safe %>
+    <!-- 'govuk_component/page-title' -->
+    <div class="govuk-title">
+      <p class="govuk-title-context">
+        Topic
+      </p>
+      <h1>
+        <%= @content_item.title %>
+      </h1>
+    </div>
+
+  </div>
+  <div class="column-third">
+    <div class="govuk-community-contact">
```

Comment:
I not sure this is worth turning into a component - and we shouldn't re-use the naming convention for something that isn't.